### PR TITLE
BER - FTMA2021 - Jan Colomer

### PR DESCRIPTION
**Pull request by Jancololo**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -28,6 +28,13 @@ app.locals.title = `${capitalized(projectName)}- Generated with IronGenerator`;
 const index = require("./routes/index");
 app.use("/", index);
 
+const celebrities = require("./routes/celebrities");
+app.use("/", celebrities);
+
+const movies = require("./routes/movies");
+app.use("/", movies);
```
Comment (wassbeer) [inline app.js:35]: I would use "/movies" as the base route, and then continue based on that in movies.js.
Same goes for Celebrities.

File: routes/movies.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+const router = require("express").Router();
+const { populate } = require("../models/Celebrity");
+const Celebrity = require("../models/Celebrity");
+const Movie = require("../models/Movie");
+
+
+//see all the movies
+router.get("/movies", (req, res, next) => {
```
Comment (wassbeer) [inline routes/movies.js:8]: Then here, it would just be "/".

File: routes/movies.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  .then(() => {
+    res.redirect('/movies')
+  })
+  .catch(err => {
+    next(err);
+  })
+})
+
+//add new movie
+router.get('/movies/new', (req, res, next) => {
```
Comment (wassbeer) [inline routes/movies.js:38]: And here "/new".

**Review by wassbeer — COMMENTED** [review]
Great stuff Jan!
Please check out my comments for potential improvements.
Cheers!

**Comment by Jancololo** [comment]
Yes! that makes more sense. Thanks for the feedback!

**Pull request closed without merging**